picard/ui/matchdetailsdialog/matchdetails.py

```diff
@@ -0,0 +1,211 @@
+
+        self.show_table(selected_row_nbr=selected_row_nbr, sort_column='similarity', sort_order=QtCore.Qt.DescendingOrder)
+
+    def accept_event(self, selected_entry_nbr):
+        match_details_log.set_file_result_selected_entry_nbr(self.current_file.filename, selected_entry_nbr)
+        self.load_selection(selected_entry_nbr)
+
+    def load_selection(self, entry_nbr):
+        """Load the album corresponding to the selected track.
+
+        Allow user to pick from our new screen and use that track instead of one its currently assigned to.
+        Based on logic from searchdialog/track.py line 186
+        """
+
+        entry = self.entries[entry_nbr]
+
+        if entry.rel_id != "":
+            # The track is not an NAT
+            self.tagger.get_release_group_by_id(entry.rg_id).loaded_albums.add(entry.rel_id)
+
+            # Have to move that file from its existing album to the new one.
+            if isinstance(self.current_file.parent, Track):
+                album = self.current_file.parent.album
+                self.tagger.move_file_to_track(self.current_file, entry.rel_id, entry.rec_id)
+                if album._files == 0:
+                    # Remove album if it has no more files associated
+                    self.tagger.remove_album(album)
+            else:
+                self.tagger.move_file_to_track(self.current_file, entry.rel_id, entry.rec_id)
+        else:
+            if self.current_file and getattr(self.current_file.parent, 'album', None):
+                album = self.current_file.parent.album
+                self.tagger.move_file_to_nat(self.current_file, entry.rec_id, None)
+                if album._files == 0:
+                    self.tagger.remove_album(album)
+            else:
+                self.tagger.load_nat(entry.rec_id, None)
+                self.tagger.move_file_to_nat(self.current_file, entry.rec_id, None)
+
+        log.debug("using alternative release id:" + entry.rel_id)
+
+    def export_match_dtl_entries_to_file(self):
+        """ Export entries to a file that can be viewed in external program
+        """
+        try:
+            # Prompt user for file location to save to
+            dir = str(Path.home())
+            suggested_file_and_path = dir + os.sep + "picard_match_results.tsv"
+            save_file = QFileDialog.getSaveFileName(None, 'Save File', suggested_file_and_path)
+            if save_file[0] == '':
+                return
+            else:
+                filename = save_file[0]
+
+            # Alternatively write to temp dir:
+            #filename = tempfile.gettempdir() + os.path.sep + "picard_match_details.tsv"
+
+            with open(filename, "w") as text_file:
+                print(match_details_log.MatchDtlEntry.hdr_tsv, file=text_file)
+
+                for row, entry in enumerate(self.entries):
+                    output_str = str(row) + "\t" + entry.fmtd_as_tsv + os.linesep
```

Comment:
Not sure we want the file format to be system-dependent.

Reply:
do you mean the line separators?  I can't think of a reason why you would want to hardcode it, as it would be wrong for some os.

Reply:
What I mean is that you could use `\n` instead of variable `os.linesep` so the format is the same on all platforms.
But if we don't care, you may as well use:

```python
print('%d\t%s' % (row, entry.fmtd_as_tsv), file=text_file)
```

And don't bother with `os.linesep`